magento/magento2
# Fixed issue Add to Wishlist with product custom option file upload not working Magento 2.4.6 #37437

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#37437
### Manual testing scenarios (*)

1. Create a simple product
2. Add a custom option file and text
3. Options should be required.
![234768126-aedf846c-916e-4ef7-912b-19825ddb1d54](https://user-images.githubusercontent.com/85498741/235427755-fee75a79-9531-4688-84e2-e5f49e515427.png)
4. Now go to the front end, log in customer, and open the product page.
5. Upload file and add text value
6. Now click on add to wishlist

### Questions or comments


### Contribution checklist (*)
